composer.json

```diff
@@ -33,7 +33,7 @@
         "cocur/slugify": "^4.0",
         "composer/package-versions-deprecated": "1.11.99.5",
         "demos-europe/edt-extra": "^0.24.0",
-        "demos-europe/demosplan-addon": "^0.24",
+        "demos-europe/demosplan-addon": "dev-ADO_t17195_sprint_cockpit_update",
```

Comment:
This should be adjusted to an new demosplan-addon version with the required changes.